restcomm/restcomm.testsuite/src/test/java/org/mobicents/servlet/restcomm/http/RestcommGeolocationsTool.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * TeleStax, Open Source Cloud Communications
+ * Copyright 2011-2016, Telestax Inc and individual contributors
+ * by the @authors tag.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+        
+package org.mobicents.servlet.restcomm.http;
+
+import java.io.IOException;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.WebResource;
+import com.sun.jersey.api.client.filter.HTTPBasicAuthFilter;
+
+/**
+ * @author fernando.mendioroz@telestax.com (Fernando Mendioroz)
+ *
+ */
+public class RestcommGeolocationsTool {
+
+    private static RestcommGeolocationsTool instance;
+    private static String geolocationsUrl;
+
+    public static RestcommGeolocationsTool getInstance() {
+        if (instance == null) {
+            instance = new RestcommGeolocationsTool();
+        }
+        return instance;
+    }
+
+    private String getGeolocationsUrl(String deploymentUrl, String accountSid, Boolean xml) {
+        if (deploymentUrl.endsWith("/")) {
+            deploymentUrl = deploymentUrl.substring(0, deploymentUrl.length() - 1);
+        }
+        geolocationsUrl = deploymentUrl + "/2012-04-24/Accounts/" + accountSid + "/Geolocation";
```

Comment:
Again, this is an adaptation of reused software. However, I agree with you in the constant for API version and account and replaced it instead of this: "/2012-04-24/Accounts/" in that string. Thanks @jaimecasero